### PR TITLE
[PLAYER-2960] Added logic for multi audio and CC

### PR DESCRIPTION
**Pull request by MarkEPAM**

### CODE REVIEW

File: sdk/react/panels/AudioAndCCSelectionPanel.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
+/**
+ * Sample React Native App
```
Comment (IvanSakharovskii) [inline sdk/react/panels/AudioAndCCSelectionPanel.js:2]: Change comment for this file

File: sdk/react/panels/AudioAndCCSelectionPanel.js [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+  },
+
+  renderHeaderView: function(hasMultiAudioTracks, hasClosedCaptions) {
+    var leftTitle;
+    var rightTitle;
+
+    var localizedAudioTitle = Utils.localizedString(this.props.config.locale, "Audio", this.props.config.localizableStrings);
+    var localizedSubtitlesTitle = Utils.localizedString(this.props.config.locale, "Subtitles", this.props.config.localizableStrings);
+
+    if (hasMultiAudioTracks && hasClosedCaptions) {
```
Comment (IvanSakharovskii) [inline sdk/react/panels/AudioAndCCSelectionPanel.js:91]: set default values to left and right title, and create 2 separate `if`, first for closed captions, second for multi audio

**Review by M0rtyMerr — APPROVED** [review]

File: sdk/react/panels/AudioAndCCSelectionPanel.js [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+  renderCCSelectionScrollView: function() {
+    var selectedClosedCaptionsLanguage = this.props.selectedClosedCaptionsLanguage;
+    var offButtonTitle = Utils.localizedString(this.props.config.locale, "Off", this.props.config.localizableStrings);
+
+    if (!this.props.closedCaptionsLanguages || this.props.closedCaptionsLanguages[0] !== offButtonTitle) {
+      this.props.closedCaptionsLanguages.splice(0, 0, offButtonTitle)
+    }
+
+    if (!selectedClosedCaptionsLanguage || selectedClosedCaptionsLanguage == offButtonTitle || selectedClosedCaptionsLanguage == "") {
```
Comment (IvanSakharovskii) [inline sdk/react/panels/AudioAndCCSelectionPanel.js:144]: Use `===` for comparing.

Comment (MarkEPAM) [reply sdk/react/panels/AudioAndCCSelectionPanel.js:144]: "==" is better than "===" in this case (string)

File: sdk/react/panels/AudioAndCCSelectionPanel.js (hunk 2 above)
Comment (MarkEPAM) [reply sdk/react/panels/AudioAndCCSelectionPanel.js:91]: it's necessary logic for titles

**Pull request merged**